Source/AssetRipper.Processing/AnimatorControllers/VirtualAnimationFactory.cs

```diff
@@ -410,12 +416,16 @@ private static IAnimatorState CreateAnimatorState(ProcessedAssetCollection virtu
 			for (int i = 0; i < Transition.ConditionConstantArray.Count; i++)
 			{
 				ConditionConstant conditionConstant = Transition.ConditionConstantArray[i].Data;
-				if (conditionConstant.ConditionMode != (int)AnimatorConditionMode.ExitTime)
+				if (!(Transition.Has_ExitTime() && conditionConstant.ConditionMode != (int)AnimatorConditionMode.ExitTime))
```

Comment:
What was the reasoning for this? It inverts the logic that was used before.

Before:
```
Runs when condition mode is not ExitTime
```
Now:
```
Runs when `Transition` doesn't have the `ExitTime` property or condition mode is ExitTime
```